src/command_modules/azure-cli-interactive/azure/cli/command_modules/interactive/__init__.py

```diff
@@ -19,7 +19,13 @@
 class InteractiveCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
-        super(InteractiveCommandsLoader, self).__init__(cli_ctx=cli_ctx)
+        from azure.cli.core import ModExtensionSuppress
+        super(InteractiveCommandsLoader, self).__init__(
+            cli_ctx=cli_ctx,
+            suppress_extension=ModExtensionSuppress(
+                __name__, 'alias', '0.5.1',
```

Comment:
This isn't clear to me--does this suppress ONLY 0.5.1?  Why is interactive suppressing alias?  What happens if I have alias with the bad version and try to run it?